tools/wptrunner/wptrunner/browsers/sauce_setup/safari-prerun.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
+curl https://raw.githubusercontent.com/w3c/web-platform-tests/master/fonts/Ahem.ttf > ~/Library/Fonts/Ahem.ttf
```

Comment:
Can we download files over the tunnel? Does the tunnel already exist at the point the prerun script runs? Because if it does, then we could just download http://web-platform.test:8000/fonts/Ahem.ttf, which would be the right version for the revision being run?

Reply:
Sure, that might be a better appraoch. This one took like 5 minutes to implement and would therefore be good to land in the meantime if only it actually worked  :)